src/Workspaces/Core/MSBuild/MSBuild/Build/ProjectBuildManager.cs

```diff
@@ -87,7 +87,10 @@ public ProjectBuildManager(ImmutableDictionary<string, string> additionalGlobalP
 
                 using (var stream = FileUtilities.OpenAsyncRead(path))
                 using (var readStream = await SerializableBytes.CreateReadableStreamAsync(stream, cancellationToken).ConfigureAwait(false))
+#pragma warning disable CA3075 // Insecure DTD processing in XML
+                // s_xmlReaderSettings is compliant with CA3075 (no resolver, prohibit DTD) but the analyzer doesn't pick it up correctly
                 using (var xmlReader = XmlReader.Create(readStream, s_xmlReaderSettings))
```

Comment:
📝 This is not a performance-sensitive code path for a single allocation. I would prefer to see the value inlined so the analyzer does not need to be suppressed.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/2105

Reply:
Since this PR no longer introduces the analyzer, I'll leave this. Its easy enough to address when the analyzer is put in place if https://github.com/dotnet/roslyn-analyzers/issues/2105 has not been resolved first.